### PR TITLE
Remove walkthrough experiment and update the Help links to point to it

### PR DESCRIPTION
**Pull request by bobbrow**
This reverts #10812, fixes a string, and enables the walkthrough for all users.

(note: I'm running into some issues with getting the commands to work as expected, so opening the PR in draft mode for now)

### CODE REVIEW

**Review by Colengms — DISMISSED** [review]

**Comment by bobbrow** [comment]
Works around https://github.com/microsoft/vscode/issues/187958 now.

File: Extension/src/LanguageServer/client.ts [hunk 1]
```diff
@@ -1048,17 +1049,39 @@ export class DefaultClient implements Client {
...
-                        return;
+                        step = step + "mac";
+                        break;
                     default: // Linux
-                        void vscode.commands.executeCommand('vscode.open', "https://go.microsoft.com/fwlink/?linkid=2217615");
-                        return;
+                        step = step + "linux";
+                        break;
                 }
+                void vscode.commands.executeCommand(
```
Comment (sean-mcmanus) [inline Extension/src/LanguageServer/client.ts:1074]: It seems like this should be `return` instead of `void`, right?

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

File: Extension/src/LanguageServer/client.ts (hunk 1 above)
Comment (Colengms) [reply Extension/src/LanguageServer/client.ts:1074]: Should it?  Seems like we don't need to pause our own execution waiting for VS Code to process this command.

Comment (sean-mcmanus) [reply Extension/src/LanguageServer/client.ts:1074]: I don't see how `return` pauses our execution. Wouldn't that be `await`?

Comment (sean-mcmanus) [reply Extension/src/LanguageServer/client.ts:1074]: Oh, you mean if VS Code itself awaited the result? Okay, sure, `void` makes sense then.

**Review by sean-mcmanus — APPROVED** [review]

File: Extension/src/LanguageServer/client.ts (hunk 1 above)
Comment (sean-mcmanus) [reply Extension/src/LanguageServer/client.ts:1074]: Or should we use `return` in case the caller actually wants to wait for the command to execute?

Comment (bobbrow) [reply Extension/src/LanguageServer/client.ts:1074]: What I think is confusing is that some paths in this function return the promise from the last function call and others don't. Prior to #11085, none of these awaitable functions were awaited nor were their promises returned.  I reverted a change that was made before #11085.

EDIT: It doesn't look like the pattern established was to return the promises from `vscode.commands.executeCommand` in this function.

**Review by Colengms — APPROVED** [review]

**Pull request merged**